src/jdbc/cubrid/jdbc/jci/UStatement.java

```diff
@@ -128,7 +129,7 @@
 
 	public int result_cache_lifetime;
 	private boolean result_cacheable = false;
-	private UStmtCache stmt_cache;
+	public UStmtCache stmt_cache;
```

Comment:
how about creating a getter for it and leave it to private.

Reply:
Ok, I will revert to private.